### ISSUE TITLE
Question: Getting error about docker-compose version mismatch, any ideas? 

### ISSUE THREAD

**Issue by neilwheeler** [jfroment/seedbox#32]
When running ./run-seedbox.sh I'm getting this error

ERROR: Version mismatch: file ./docker-compose.yaml specifies version 3.8 but extension file ./services/traefik.yaml uses version 1

Any ideas what might be the cause?

**Comment by jfroment** [comment]
Hello,

What version of docker-compose are you running? Does it meet the >= 2.2 criteria?
If not, I invite you to update your system package or use a direct installation using the binary from here: https://github.com/docker/compose/releases

**Comment by neilwheeler** [comment]
Yes I had the latest v2 version installed. The thing that's confusing me is that the [documentation](https://github.com/docker/compose/blob/v2/README.md) for compose v2 states that the command is `docker compose` as opposed to `docker-compose` in v1. But your run_seedbox.sh script uses `docker-compose`. 
I must be missing something here but I don't know what!

**Comment by FlyingKawasaki74** [comment]
> Yes I had the latest v2 version installed. The thing that's confusing me is that the [documentation](https://github.com/docker/compose/blob/v2/README.md) for compose v2 states that the command is `docker compose` as opposed to `docker-compose` in v1. But your run_seedbox.sh script uses `docker-compose`. I must be missing something here but I don't know what!

Can confirm, I also had the problem that `docker-compose` which is used in run-seedbox.sh was not recognized.
Environment is Ubuntu 22, I set up the newer versions of Docker and Docker Compose according to https://docs.docker.com/engine/install/ubuntu/
I solved this by renaming the two occurences in run-seedbox.sh

**Comment by jfroment** [comment]
I think I understood it. Personally I installed docker compose v2 (latest) not by using a package manager but directly on github (grabbed the binary directly), and named it "docker-compose" in /usr/local/bin. I named it out of habit, and then developed my scripts around it.
So my bad, I name it differently. I will add an option in the script to make it variable, so everyone would get the possibility to use its own version and/or name.
